kotest-assertions/kotest-assertions-json/src/commonTest/kotlin/io.kotest.assertions.json/schema/ArraySchemaTest.kt

```diff
@@ -102,5 +102,81 @@ class ArraySchemaTest : FunSpec(
             $ => Sequence should be Unique
          """.trimIndent()
       }
+
+      test("Array not contains string") {
+         val array = "[1,1]"
+         val containsStringArray = jsonSchema {
+            array(contains = containsSpec { string() })
+         }
+         "[\"bob\"]" shouldMatchSchema  containsStringArray
+         shouldFail { array shouldMatchSchema containsStringArray }.message shouldBe """
+            $ => Expected any item of type string
+            $.contains[0] => Expected string, but was number
+            $.contains[1] => Expected string, but was number
```

Comment:
```suggestion
            $ => Expected some item to match contains-specification:
                $[0] => Expected string, but was number
                $[1] => Expected string, but was number
```

Perhaps, to make it a bit more distinguished in case we get it in an object with other properties as well. What do you think?